faq.md

```diff
@@ -59,13 +60,14 @@ software.
 
 If the license allows the recipient to [use, study, share and
 improve](https://fsfe.org/freesoftware/basics/4freedoms.html) the software, then
-that software is Free Software.
+that software is [Free Software](https://www.gnu.org/philosophy/free-sw.html).
 
 ## Which license should I choose? {#which-license}
 
-Ultimately, the license you choose is up to you. If you are contributing to an
-existing project, you should release your changes under the same license as the
-project. Otherwise, the [Free Software
+Always choose a Free Software license, i.e., a license that gives the recipient
+the four freedoms mentioned above. Aside from that, the license you choose is up
```

Comment:
"mentioned above" refers to something found on the linked website, which may be a bit confusing for people who don't click on every link. I would probably use the wording "... a license that gives the recipient the freedom to use, study, share, and improve the software".